.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Nitpick: probably don't want to check in this VSCode file

Reply:
Actually this is a project configuration file for VSCode. I generally like to check these in as it ensures anybody using VSCode has the same settings applied to the project automatically.

Reply:
Ah ok cool! Didn't realize it was intentional. Sounds good to me.